department-of-veterans-affairs/caseflow
# Added safe handling of form diffing

Resolves #14171

### Description
Ensures that both first and second forms are present before attempting diff

### Acceptance Criteria

### Testing Plan
1. Select judge "BVAYELLOW": http://localhost:3000/test/users
1. Navigate to hearings:  http://localhost:3000/hearings/schedule
1. Choose any hearing
1. Click the "Edit hearing details" link
1. Ensure that there are no errors changing the virtual hearing between video/virtual
